src/config/media/pipewire.md

```diff
@@ -0,0 +1,64 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+[pipewire(1)](https://man.voidlinux.org/pipewire) should be started as a user.
+Use the autostarting mechanism of your desktop environment or
+[startx](../graphical-session/xorg.md#startx).
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+`/etc/pipewire/pipewire.conf` for auto-starting the PulseAudio server, it's
+recommended to start it from the same place where you start PipeWire by
+executing `pipewire-pulse`. This keeps PipeWire's configuration file unmodified
+for smoother future upgrades.
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and reboot.
```

Comment:
Reboot sounds like the wrong advice. I think this should be worded to "make sure no other PulseAudio server instance is running".